spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -77,9 +77,16 @@ fun calculatePackageVerificationCode(files: List<File>, excludes: List<String> =
  * Return the SHA-1 sum of the given file as hex string.
  */
 private fun sha1sum(file: File): String {
-    file.inputStream().use {
+    file.inputStream().use { inputStream ->
```

Comment:
Commit message: You might want to refer to commit 86ffebb78952eef94f6c5a0d247a47680af17a49 in which I explained that processing of large files > 2 GB would also require a chunked approach. So, you commit not only reduces memory use, but also allow for processing of files > 2 GB in general, independent of memory usage.

Reply:
Done.